### PR TITLE
Create a slow queue for tasks that require locking on git or Github

### PR DESCRIPTION
**Pull request by bdunne**
img_20151203_134528552_hdr

### CODE REVIEW

**Comment by bdunne** [comment]
@Fryguy Please review

**Comment by bdunne** [comment]
cc @matthewd @jrafanie

File: Procfile.tmpl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 thin:            bundle exec rails s -p $PORT
 sidekiq:         bundle exec sidekiq -q miq_bot
+sidekiq:         bundle exec sidekiq -q miq_bot_glacial
```
Comment (matthewd) [inline Procfile.tmpl]: Doesn't this name (lhs) need to be unique?

Comment (Fryguy) [inline Procfile.tmpl]: It doesn't *have* to be, but it makes reading the log much harder if it's not.  @bdunne Please change.

Comment (jrafanie) [inline Procfile.tmpl]: Yes, good call

**Comment by Fryguy** [comment]
:+1: aside from the foreman name thing.

**Comment by jrafanie** [comment]
I can't wait to see glacial log messages

**Comment by jrafanie** [comment]
Looks good to me.

**Pull request merged**